cluster/engine.go

```diff
@@ -1015,12 +1015,28 @@ func (e *Engine) Pull(image string, authConfig *types.AuthConfig) error {
 
 // Load an image on the engine
 func (e *Engine) Load(reader io.Reader) error {
-	err := e.client.LoadImage(reader)
+	loadResponse, err := e.apiClient.ImageLoad(context.TODO(), reader, false)
```

Comment:
We've been using `context.TODO()` for all `engine-api` calls so far. What do you recommend?